src/pretix/presale/templates/pretixpresale/event/base.html

```diff
@@ -22,16 +22,16 @@
 {% endblock %}
 {% block page %}
     <div class="page-header">
-        <h1 class="pull-left">
+        <div class="pull-left">
             {% if event_logo %}
                 <a href="{% eventurl event "presale:event.index" %}" title="{{ event.name }}">
                     <img src="{{ event_logo|thumbnail_url:'logo' }}" alt="{{ event.name }}" class="event-logo" />
                 </a>
             {% else %}
-                <a href="{% eventurl event "presale:event.index" %}">{{ event.name }}</a>
+                <h1><a href="{% eventurl event "presale:event.index" %}">{{ event.name }}</a></h1>
                 <small>{{ event.get_date_range_display }}</small>
```

Comment:
Was this intended? Before, the ``<small>`` was a sub-element of the ``<h1>``. Let's keep it this way for now. (It doesn't look that bad in the line below, but we'd need to adjust the language switcher then.)